### PR TITLE
test

### PR DESCRIPTION
**Pull request by WingT**
Change-Id: Iacfcde9e73330360a4de01f9b3916abac7f1a3e9

### CODE REVIEW

File: hot.c [hunk 1]
```diff
@@ -13,6 +13,7 @@ int main(int argc, char *argv[]){
   if (argc == 2)
     parallel = atoi(argv[1]);
   pthread_t *threads;
+  //threads = malloc(sizeof(pthread_t) * parallel);
```
Comment (WingT) [inline hot.c]: why comment?

File: hot.c [hunk 2]
```diff
@@ -13,6 +13,7 @@ int main(int argc, char *argv[]){
   if (argc == 2)
     parallel = atoi(argv[1]);
   pthread_t *threads;
+  /*threads = malloc(sizeof(pthread_t) * parallel);*/
```
Comment (WingT) [inline hot.c]: why comment again

File: hot.c [hunk 3]
```diff
@@ -13,6 +13,7 @@ int main(int argc, char *argv[]){
...
   threads = malloc(sizeof(pthread_t) * parallel);
   for (int i = 0; i < parallel; i++){
     int result_code = pthread_create(&threads[i], NULL, worker_fun, NULL);
```
Comment (WingT) [inline hot.c]: good

File: hot.c [hunk 4]
```diff
@@ -13,6 +13,7 @@ int main(int argc, char *argv[]){
   if (argc == 2)
     parallel = atoi(argv[1]);
```
Comment (vossem) [inline hot.c]: 666

File: hot.c [hunk 5]
```diff
@@ -13,6 +13,7 @@ int main(int argc, char *argv[]){
   if (argc == 2)
```
Comment (vossem) [inline hot.c]: test

Comment (vossem) [inline hot.c]: change

File: hot.c [hunk 6]
```diff
@@ -13,6 +13,7 @@ int main(int argc, char *argv[]){
...
     parallel = atoi(argv[1]);
   pthread_t *threads;
```
Comment (WingT) [inline hot.c]: test

**Pull request closed without merging**